staging/src/k8s.io/kms/go.sum

```diff
@@ -1,34 +1,49 @@
+cloud.google.com/go/compute v1.21.0/go.mod h1:4tCnrn48xsqlwSAiLf1HXMQk8CONslYbdiEZc9FEIbM=
```

Comment:
changes in this file are surprising, right?

Reply:
@dims says it because of otel/grpc 😭

Reply:
all i do is run `hack/update-vendor.sh` again and again and 😢

Reply:
the direct and indirect deps in the kms go.mod file still look fine to me

Reply:
but it's *super* weird that touching the _mock subdir messed with this go.sum :-/